mne/annotations.py

```diff
@@ -880,6 +910,24 @@ def _check_event_id(event_id, raw):
         raise ValueError('Invalid input event_id')
 
 
+def _check_event_description(event_desc, events):
+    """Check event_id and convert to default format."""
+    # check out event_id dict
+    if event_desc is None:  # convert to int to make typing-checks happy
+        event_desc = list(np.unique(events[:, 2]))
+
+    if isinstance(event_desc, dict):
+        for val in event_desc.values():
+            _validate_type(val, (str, None), 'Event names')
+    elif isinstance(event_desc, list):
+        event_desc = dict(zip(event_desc, (str(i) for i in event_desc)))
+    elif callable(event_desc):
+        pass
+    else:
```

Comment:
can you add a comment to explain what type you catch here?

Reply:
I removed this, we shouldn't hit this case.